builder/tests/configbean/src/test/java/io/helidon/builder/config/test/BasicConfigBeanTest.java

```diff
@@ -149,4 +150,38 @@ void noConfig() {
                    equalTo(List.of()));
     }
 
+    @Test
+    void equality() {
+        Config cfg = Config.builder()
+                .sources(ConfigSources.classpath("io/helidon/builder/config/test/BasicConfigBeanTest.yaml"))
```

Comment:
kebab-case is more conventional for yaml file name

Reply:
I noticed that the majority of test cases use the convention I am using - I have no problem making this change obviously - but wanted to double-check.

Reply:
please switch to kebab-case